tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/DataType/DataType.spec.ts

```diff
@@ -15,19 +17,21 @@ test.describe('DataType tests', () => {
   ];
 
   test.beforeEach(async ({umbracoApi}) => {
-    await umbracoApi.dataType.ensureNameNotExistsAtRoot(dataTypeName);
+    await umbracoApi.dataType.ensureNameNotExists(dataTypeName);
+    await umbracoApi.dataType.ensureNameNotExists(folderName);
```

Comment:
```suggestion
```
Since we do not create a folder in most of the tests. I don't think we ensure it does not exist before/after each test